src/redisraft.c

```diff
@@ -1166,9 +1171,8 @@ __attribute__((__unused__)) int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisMod
         return REDISMODULE_ERR;
     }
 
-    RedisModule_Log(ctx, REDIS_NOTICE, "RedisRaft version %s [%s]",
-            REDISRAFT_VERSION,
-            REDISRAFT_GIT_SHA1);
+    RedisModule_Log(ctx, "notice", "RedisRaft version %s [%s]",
```

Comment:
it might be a little verbose, but why not use the #defines from redismodule.h

i.e.

```
#define REDISMODULE_LOGLEVEL_DEBUG "debug"
#define REDISMODULE_LOGLEVEL_VERBOSE "verbose"
#define REDISMODULE_LOGLEVEL_NOTICE "notice"
#define REDISMODULE_LOGLEVEL_WARNING "warning"
```

(and in other usages below)